testing-guidance/workflow-tests/Viewport/viewport-workflow-tests.md

```diff
@@ -0,0 +1,77 @@
+﻿# Viewport Workflow Tests
+​
+Testing in this area should focus on basic Viewport functionality.
+​
+## Common Issues to Watch For
+​
+Test guidance will sometimes note specific issues to watch for. The common issues below should be watched for through all testing, even if unrelated to the current workflow being tested.
+
+1. Console log errors/warnings/spam
+2. Asserts
+​
+### Platforms:​
+
+- Windows
+- Linux​
+​
+### Documentation pages:​
+
+* [Working with the 3D Viewport](https://www.o3de.org/docs/user-guide/editor/Viewport/)
+* [Reference Spaces](https://www.o3de.org/docs/user-guide/editor/Viewport/reference-spaces/)
+* [Group Selection](https://www.o3de.org/docs/user-guide/editor/Viewport/group-selection/)
+* [Advanced 3D Viewport Operations](https://www.o3de.org/docs/user-guide/editor/Viewport/advanced-operations/)​
+​
+### Workflow 1: Camera control 
+​
+**Product:** 
+A Level with multiple entities with a mesh component and assigned shaderball model asset.
+​
+**Suggested Time Box:** 5-10 mins
+
+| Workflow                                                               | Requests																																																																						   | Things to Watch For                                                                                                                                                          |
+|------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Navigate level with camera controls									 | <ol><li>Select the Viewport.</li><li>Navigate the camera with `W`, `A`, `S`, `D`, `Q`, `E`. Hold `shift` for fast move.</li><li>`Right-click and drag` to rotate the camera.</li><li>Press `MMB and drag` to pan up and down.</li><li>Scroll up and down using the `MMB`, to zoom in and out.</li><li>`ALT+LMB` to orbit.</li></ol>| <ul><li>Correct camera behaviour.</li></ul>																													  |
```

Comment:
suggestion: It would be good to mention using Pan/Scroll while also holding Alt too (as these behaviors also work while orbiting the camera). It might also be worth mentioning testing clicking in and out of the viewport while moving the camera, as in the past we have had a few subtle issues with focus causing problems when clicking in and out of the viewport.